query/src/main/java/org/infinispan/query/impl/InfinispanQueryStatisticsInfo.java

```diff
@@ -17,13 +17,6 @@
 public class InfinispanQueryStatisticsInfo implements InfinispanQueryStatisticsInfoMBean {
 
```

Comment:
In that case please remove the unused imports, also make getObjectLoadingExecutionAvgTime just delegate to sf.getStatistics().getObjectLoadingExecutionAvgTime() instead of actually doing the computation.
Also fix the javadoc, which no longer reflects the purpose of these classes.  Or just remove them entirely? We can add them when needed :)